src/main/java/javaslang/control/ValidationBuilder.java

```diff
@@ -0,0 +1,75 @@
+/*     / \____  _    _  ____   ______  / \ ____  __    _ _____
+ *    /  /    \/ \  / \/    \ /  /\__\/  //    \/  \  / /  _  \   Javaslang
+ *  _/  /  /\  \  \/  /  /\  \\__\\  \  //  /\  \ /\\/  \__/  /   Copyright 2014-now Daniel Dietrich
+ * /___/\_/  \_/\____/\_/  \_/\__\/__/___\_/  \_//  \__/_____/    Licensed under the Apache License, Version 2.0
+ */
+package javaslang.control;
+
+import javaslang.Function2;
+import javaslang.Function3;
+import javaslang.Function4;
+import javaslang.collection.List;
+
+public class ValidationBuilder<E,T1,T2> {
```

Comment:
Please

* move the ValidationBuilder* into the Validation interface. I see them as '2nd level API', they should not be accessed directly, we use Validation API instead (some kind of 'single point of contact').
* 'flatten' the ValidationBuilders, i.e. Validation.ValidationBuilder, Validation.ValidationBuilder3, ...
* make the classes final
* make the constructors private